packages/cli/src/execute/handler.ts

```diff
@@ -67,6 +69,14 @@ const executeHandler = async (options: ExecuteOptions, logger: Logger) => {
           { packages: autoInstallTargets, repoDir },
           logger
         );
+
+        // create a module resolution dictionary.
+        // this is to map aliases like @latest & @next to what they resolved into
+        if (autoInstallTargets.length === options.adaptors.length) {
```

Comment:
This is expected to always be equal. 
Because after installation. The runtime returns list of all resolved adaptors which is a one-to-one mapping to the list of installTargets passed to it.